Documentation/Dockerfile

```diff
@@ -2,26 +2,16 @@ FROM docker.io/library/python:3.7.5-alpine3.10
 
 LABEL maintainer="maintainer@cilium.io"
 
-ENV DOCS_DIR=/srv/Documentation
-ENV API_DIR=/srv/api
-
-ADD ./requirements.txt $DOCS_DIR/requirements.txt
-
-ENV PACKAGES="\
+RUN apk add --no-cache --virtual --update \
+    aspell-en \
     bash \
     ca-certificates \
-    make \
+    enchant \
```

Comment:
Is this distinct from pyenchant, which should be installed with the requirements?

Reply:
Good question... I haven't questioned these dependencies, keeping them as they are for the purpose of this PR :)